release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -127,6 +128,21 @@ You can use a REST API to develop applications to consume and respond to events
 ====
 This feature is supported for single node OpenShift clusters only.
 ====
+=======
```

Comment:
@skrthomas I think this line was inadvertent? It throws off the level headings in the file, so I'm removing it as part of https://github.com/openshift/openshift-docs/pull/41265